### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kabilar**
- Add citation section
- Add links to elements.datajoint.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
 # DataJoint Element - DeepLabCut
 
-This repository features a DataJoint schema for pose estimation via ***DeepLabCut***.
++ This repository features a DataJoint schema for pose estimation via ***DeepLabCut***.
```
Comment (CBroz1) [inline README.md:3]: Do we have a prescriptive design doc for what should be bullets vs. prose? My gut is that something > 2 sentences shouldn't be bulleted, so I left these as paragraphs. I would propose removing bullets for these first 2 paragraphs, but ignore if this contradicts design docs elsewhere

Comment (CBroz1) [inline README.md:3]:
```suggestion
This repository features a DataJoint schema for pose estimation via ***DeepLabCut***.
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 The pipeline presented here is not a complete pipeline by itself, but rather a modular
 design of tables and dependencies specific to the behavior tracking workflow. This
 modular pipeline element can be flexibly attached downstream to any particular design
 of experiment session, thus assembling a fully functional behavior pipeline (see the
 example [workflow-deeplabcut](https://github.com/datajoint/workflow-deeplabcut)).
 
-This Element currently supports single-animal, single-camera 2D models, and does not yet
++ This Element currently supports single-animal, single-camera 2D models, and does not yet
```
Comment (CBroz1) [inline README.md:10]:
```suggestion
This Element currently supports single-animal, single-camera 2D models, and does not yet
```

**Review by CBroz1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kabilar) [reply README.md]: Thanks @CBroz1.  We do not currently have a design doc.  I generally lean to using bullet points in READMEs, but am happy to apply your suggestions here.

**Review by CBroz1 — APPROVED** [review]

**Pull request merged**